### PR TITLE
Feature/adji

### PR DESCRIPTION
**Pull request by adjifsyah**
Add searchController and tableView in AFViewController

### CODE REVIEW

**Comment by lutluthfi** [comment]
please provide the screenshot or videos

File: README.md [hunk 1]
```diff
@@ -17,3 +17,6 @@
 **AT** = Nuzulul Athaya  
 **YA** = Yuda Aqil Siraj  
 **AF** = Adji Firmansyah
+
+![Simulator Screen Shot - iPhone 12 Pro Max - 2022-03-21 at 08 21 50](https://user-images.githubusercontent.com/65451696/159194930-5851da00-1cce-4a29-b3f2-addddea9d217.png)
+![Simulator Screen Shot - iPhone 12 Pro Max - 2022-03-21 at 08 22 24](https://user-images.githubusercontent.com/65451696/159194938-09d683ed-bff2-4d01-9ded-90899da2bda8.png)
```
Comment (lutluthfi) [inline README.md:22]: screenshot nya di description PR ya, bukan di README.md

Comment (adjifsyah) [reply README.md:22]: siap kak, saya kira di README.md 😅

Comment (lutluthfi) [reply README.md:22]: @adjifsyah yang di `README.md` dihapus yah

Comment (adjifsyah) [reply README.md]: sudah kak

**Review by lutluthfi — APPROVED** [review]

**Pull request merged**